config/fcos/v1_6_exp/translate_test.go

```diff
@@ -1637,3 +1637,104 @@ func TestTranslateGrub(t *testing.T) {
 		})
 	}
 }
+
+func TestTranslateSelinux(t *testing.T) {
+	cmdToExecute := "/usr/sbin/semodule -i" + "/etc/selinux/targeted/modules/active/extra/some_name.cil"
+	translations := []translate.Translation{
+		{From: path.New("yaml", "version"), To: path.New("json", "ignition", "version")},
+		{From: path.New("yaml", "selinux", "module"), To: path.New("json", "storage", "filesystems")},
+		{From: path.New("yaml", "selinux", "module"), To: path.New("json", "storage", "filesystems", 0)},
+		{From: path.New("yaml", "selinux", "module"), To: path.New("json", "storage", "filesystems", 0, "path")},
+		{From: path.New("yaml", "selinux", "module"), To: path.New("json", "storage", "filesystems", 0, "device")},
+		{From: path.New("yaml", "selinux", "module"), To: path.New("json", "storage", "filesystems", 0, "format")},
+		{From: path.New("yaml", "selinux", "module"), To: path.New("json", "storage")},
```

Comment:
Related to the other comment, we should be able to remove all of these.

Reply:
All the filesystem stuff, right?

Reply:
Yup!